### PR TITLE
Add support for recovery flashable zip

### PR DESCRIPTION
**Pull request by althafvly**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -15,31 +15,31 @@ jobs:
     - name: Build (AArch64)
       run: |
         ANDROID_NDK_HOME=./android-ndk-r23b CXX=aarch64-linux-android31-clang++ ./build.sh
-        zip -r ih8sn-aarch64.zip 60-ih8sn.sh ih8sn ih8sn.conf ih8sn.rc push.ps1 push.sh
+        zip -r ih8sn-aarch64.zip META-INF system push.ps1 push.sh
```
Comment (luk1337) [inline .github/workflows/build.yml:18]: This will put all user configs in flashable zip, not sure if this is a good idea...

Comment (althafvly) [reply .github/workflows/build.yml:18]: Releases are done on a fresh clone right?.

Comment (luk1337) [reply .github/workflows/build.yml:18]: I'm mostly referring to local builds here.

**Comment by luk1337** [comment]
@althafvly I sorted your gitignore FYI, make sure you don't discard this change on next push ^^

**Comment by althafvly** [comment]
done

**Pull request merged**